libraries/AP_Scripting/docs/docs.lua

```diff
@@ -4011,7 +4008,6 @@ function networking:get_netmask_active() end
 function networking:get_ip_active() end
 
 -- visual odometry object
---@class visual_odom
```

Comment:
This is incorrect markup. It should be `---@class`, however the class mark up is for user data objects only and should not be used on singletons. The error was overlooked due to the typo.